modules/cluster-logging-collector-log-forward-kafka.adoc

```diff
@@ -67,7 +67,7 @@ spec:
 <4> Specify a name for the output.
 <5> Specify the `kafka` type.
 <6> Specify the URL and port of the Kafka broker as a valid absolute URL, optionally with a specific topic. You can use the `tcp` (insecure) or `tls` (secure TCP) protocol. If the cluster-wide proxy using the CIDR annotation is enabled, the output must be a server name or FQDN, not an IP address.
-<7> If using a `tls` prefix, you must specify the name of the secret required by the endpoint for TLS communication. The secret must exist in the `openshift-logging` project, and must have keys of *tls.crt*, *tls.key*, and *ca-bundle.crt* that point to the respective certificates that they represent.
+<7> If you are using a `tls` prefix, you must specify the name of the secret required by the endpoint for TLS communication. The secret must exist in the `openshift-logging` project and must contain a `ca-bundle.crt` key that points to the certificate it represents.
```

Comment:
I suggest removing these words "The secret must exist in the openshift-logging project" . For mCFL, the secret isn't in Openshift-logging namespace.